config.ini

```diff
@@ -1,26 +1,40 @@
 [dpdk]
 ## Hexadecimal bitmask of cores to run on.
 lcore_mask=1
-## Port mask, enable and disable ports.
-## Default: all ports are enabled.
-#port_mask=1
 channel=4
-## Number of ports.
-nb_ports=1
 promiscuous=1
 numa_on=1
 ## TCP segment offload, default: disabled.
 tso=0
 ## HW vlan strip, default: enabled.
 vlan_strip=1
 
+# enabled port list
+#
+# EBNF grammar:
+#
+#    exp      ::= num_list {"," num_list}
+#    num_list ::= <num> | <range>
+#    range    ::= <num>"-"<num>
+#    num      ::= '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
+#
+# examples
+#    1-3       using cores 1,2,3 to handle this port
+#    1-3,4,7   using cores 1,2,3,4,7 to handle this port
+port_list=0
```

Comment:
I think this key can be commented out by default, means all ports are enabled, and we initialize it with `rte_eth_dev_count()`, this makes users more convenient to use.

Reply:
I don't think this is a  good idea, because when parse config.ini, DPDK is not initialized, so we can't call `rte_eth_dev_count` and we must postpone the initialization of `portid_list`, this  will make code complicated. I think using 0 as default value is good enough.

Reply:
Well, you are right, i forget that dpdk will not be initialized at that time.